.github/workflows/cicd.yml

```diff
@@ -208,7 +208,7 @@ jobs:
     uses: ./.github/workflows/reusable-test.yml
     with:
       name: bpf2c_fuzzer
-      test_command: .\bpf2c_fuzzer.exe bpf2c_fuzzer_corpus -use_value_profile=1 -max_total_time=900 -artifact_prefix=Artifacts\
+      test_command: .\bpf2c_fuzzer.exe bpf2c_fuzzer_corpus -use_value_profile=1 -max_total_time=300 -artifact_prefix=Artifacts\
```

Comment:
Can we make this an conditional on scheduled vs PR? Maybe run this for 15 minutes on schedule and run 5 minutes on PR?

Reply:
Done